app/services/record_declaration.rb

```diff
@@ -28,12 +28,10 @@ class RecordDeclaration
               message: I18n.t(:invalid_evidence_type),
             }
   validate :output_fee_statement_available
-  validate :validate_has_passed_field, if: :validate_has_passed?
```

Comment:
can we leave this service change out of this pr as there is another ticket to tackle this fully please?